src/test/java/org/scijava/convert/StringToNumberConverterTest.java

```diff
@@ -0,0 +1,112 @@
+
+package org.scijava.convert;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests {@link StringToNumberConverter}
+ *
+ * @author Gabriel Selzer
+ */
+public class StringToNumberConverterTest {
```

Comment:
Needs tests that non-numerically-formatted strings successfully report `false` for `canConvert`.